api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -190,7 +194,7 @@ public ResponseEntity<Profile> getMe() {
   @Override
   public ResponseEntity<Boolean> getUserTermsOfServiceStatus() {
     DbUser loggedInUser = userAuthenticationProvider.get().getUser();
-    return ResponseEntity.ok(userService.validateTermsOfService(loggedInUser));
+    return ResponseEntity.ok(userService.hasSignedLatestTermsOfServiceBoth(loggedInUser));
```

Comment:
"both" = aou + terra.  Is that obvious?

Reply:
areBothTermsOfServiceAgreed?

Reply:
I think "both" is OK, but not necessarily obvious. Maybe "all"? For me, the tricky part is remembering that we have multiple ToS since the user only sees one ToS, and I'm unsure how to reflect that in a name.

Reply:
I tweaked it slightly to be "ForBoth" - shrug.